src/radha/Shop.java

```diff
@@ -0,0 +1,95 @@
+package radha;
+
+public class Shop {
+	int maggie = 50;
+	int dosa = 43;
+	int oilPouch = 39;
+	int panipuri = 43;
+	int masala = 73;
+	
+	void requestedQty(int maggieQ, int dosaQ, int oilPouchQ, int panipuriQ, int masalaQ) {
+		if(maggieQ<=maggie) {
+			maggie = maggie-maggieQ;
+		}else{
+			System.out.println("Sorry!! Quantity available is not suffcient.");
+			System.out.println("Requested Maggie qty "+maggieQ+" is greater than qty available in stock. We have fulfilled only partial request with "+maggie+" Maggie packets.");
+			maggie=0;
```

Comment:
If request stock is not available mean we are setting direct it zero, 
what if second time your calling same method and let say you request 20 packs, then this looks not logically correct